src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1492,6 +1510,10 @@ private function getRequestInstanceForClientIpTests($remoteAddr, $httpForwardedF
         return $request;
     }
 
+    /**
+     * Kept to check backward compatibility
+     * @group legacy
```

Comment:
this does not make sense here actually. groups should be applied on tests, not on private helper methods.